ansible/roles/netbox/defaults/main.yml

```diff
@@ -583,24 +579,31 @@ netbox__config_media_root: '{{ netbox__data + "/media" }}'
 # Absolute path where Netbox reports (Python modules) are stored.
 netbox__config_reports_root: '{{ netbox__data + "/reports" }}'
                                                                    # ]]]
+# .. envvar:: netbox__config_plugins [[[
+#
+# List of Netbox plugins.
+netbox__config_plugins: []
+                                                                   # ]]]
+# .. envvar:: netbox__config_plugins_config [[[
+#
+# Netbox plugins configuration settings.
+netbox__config_plugins_config: {}
+                                                                   # ]]]
```

Comment:
Any plans to add an example in the documentation for how the plugins can be configured? I guess since there's no plugins yet that I know of, you could use the example from the official documentation.

Reply:
Makes sense. Examples are always nice.